M1_MAC_SETUP.md

```diff
@@ -0,0 +1,92 @@
+# M1 Mac Setup Guide
+
+This guide should get you set up to run `bigquery-etl` on an M1 or M2 Mac. The
```

Comment:
Are there other repositories that require this setup? Perhaps a link in the [Data onboarding documentation](https://mana.mozilla.org/wiki/display/DATA/Onboarding) would give more visibility and benefit?

Reply:
I second this, I believe this is the doc that gets distributed to joiners. I don't think we need a copy of this guide there, but maybe a disclaimer/warning on top of the doc to make all new joiners aware of general compatibility issues on Apple silicon devices so that they can avoid the setup frustrations later.